pkg/airgap/update.go

```diff
@@ -246,11 +246,13 @@ func UpdateAppFromPath(a *apptypes.App, airgapRoot string, airgapBundlePath stri
 
 func canInstall(beforeKotsKinds *kotsutil.KotsKinds, afterKotsKinds *kotsutil.KotsKinds) error {
 	var beforeSemver, afterSemver *semver.Version
-	if v, err := semver.ParseTolerant(beforeKotsKinds.Installation.Spec.VersionLabel); err == nil {
-		beforeSemver = &v
-	}
-	if v, err := semver.ParseTolerant(afterKotsKinds.Installation.Spec.VersionLabel); err == nil {
-		afterSemver = &v
+	if afterKotsKinds.License != nil && afterKotsKinds.License.Spec.IsSemverRequired {
```

Comment:
Checking the `afterKotsKinds` here as it seems we would care about the _current_ state of semver on the channel vs what it was for the last sequence. Any reason this might be problematic?

Reply:
should get the license stored in the app table instead of kots kinds, as that would be the most recent and update to date one. you can get that license using `store.GetStore().GetLatestLicenseForApp`

Reply:
pushed https://github.com/replicatedhq/kots/pull/4668/commits/d5ded03526e6bfc5d2bf90d191389733f03538e0 to get the latest license from the db